riscv/csrs.cc

```diff
@@ -620,8 +620,10 @@ reg_t misa_csr_t::dependency(const reg_t val, const char feature, const char dep
 }
 
 bool misa_csr_t::unlogged_write(const reg_t val) noexcept {
+  bool zca_changable = proc->extension_changable(EXT_ZCA);
```

Comment:
"changeable" is misspelled here and elsewhere; let's fix that before merging.

Reply:
OK. I'll fix it later.